<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(230,230,230);

}

QPushButton {
	background-color: rgb(255,255,255);  /* Background color */
    color: rgb(0, 0, 150);  /* Text color */
    border: 2px solid rgb(255,155, 50);  /* Border color */
    border-radius: 5px;  /* Rounded edges */
	padding: 2px;
}

QPushButton:hover {
    background-color: rgb(245, 245, 255);  /* Background color when hovered */
	color: rgb(0, 0, 200);
    border-color:rgb(225, 105, 0);  /* Border color when hovered */
}

QPushButton:pressed {
   background-color: rgb(200,200, 210);  /* Background color when pressed */
	border: 3px solid rgb(205, 85, 0);
}
QPushButton:clicked {
   background-color: rgb(200, 200, 210);  /* Background color when pressed */
	border:  rgb(205, 85, 0);
}

QPushButton:checked {
    background-color: rgb(220,120, 15);
	border: 3px solid rgb(205, 105, 0);
	font-weight: bold;  /* Bold text */
}

QPushButton:disabled {
    background-color: rgb(220,220, 220);
	border: 3px solid rgb(205, 105, 0);
	font-weight: bold;  /* Bold text */
}

QDoubleSpinBox:enabled{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}
QDoubleSpinBox:disabled{
color: rgb(0, 0, 200);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}

QSpinBox:enabled{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}
QSpinBox:disabled{
color: rgb(0, 0, 200);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}

QGroupBox{
color: rgb(0, 0, 150);  /* Text color */
font-weight: bold;  /* Bold text */
}

QLabel{
color: rgb(0, 0, 150);  /* Text color */
}

QPlainTextEdit{
background-color: rgb(255,255,255);
border-radius: 5px;  /* Rounded edges */
}

QLineEdit:endabled{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 3px solid rgb(255,155,50);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QLineEdit:disabled{
background-color: rgb(230,230,230);
color: rgb(0, 0, 150);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QComboBox{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 3px solid rgb(255,155,50);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QTabBar::tab {
    color: rgb(0, 0, 150); 
    border: 2px solid rgb(255,155,50);
    padding: 5px;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	min-width:80px;
}

QTabBar::tab:selected {
    color:rgb(0, 0, 150); 
    background-color: rgb(255, 255, 255); ;
    font-weight: bold;
	border: 3px solid rgb(235, 135, 30);
}

QTabBar::tab:hover {
    color: rgb(0, 0, 200); /* Slightly lighter orange */
	background-color: rgb(245, 245, 255); ;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X-Axis:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="x_world_spinbox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y-Axis:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="y_world_spinbox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>X-Cross:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="x_cross_spinbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Y-Cross:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="y_cross_spinbox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete_button">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>resources/trash.png</normaloff>resources/trash.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
